tests/validation/tests/v1_api/resource/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns-23492
+  annotations:
+    field.cattle.io/containerDefaultResourceLimit: { "limitsCpu": "0m","limitsMemory": "0Mi","requestsCpu": "0m","requestsMemory": "0Mi" }
+    "field.cattle.io/projectId": "c-pqtg5:p-krxzl"
+  labels:
+    "field.cattle.io/projectId": "p-krxzl"
```

Comment:
Isnt project id automatically generated?

Reply:
Is it a placeholder?

Reply:
yes, it is the value from where I got the template. It will be replaced by the real value in the tests.